features/attached_enable.feature

```diff
@@ -131,3 +131,135 @@ Feature: Enable command behaviour when attached to an UA subscription
             ESM Infra is already enabled.
             See: sudo ua status
             """
+
+    @series.focal
+    Scenario: Attached enable Livepatch service in a focal lxd container
+        Given a `focal` lxd container with ubuntu-advantage-tools installed
+        When I attach contract_token with sudo
+        And I run `ua enable livepatch` as non-root
+        Then I will see the following on stderr:
+            """
+            This command must be run as root (try using sudo)
+            """
+        When I run `ua enable livepatch` with sudo
+        Then I will see the following on stdout:
+            """
+            One moment, checking your subscription first
+            Cannot install Livepatch on a container
+            """
```

Comment:
To cut down on some of the test duplication, what do you think about consolidating livepatch/fips and fips-updates into a single Scenario Outline?

Maybe we can also consolidate the same non-container services tests for series.trusty too.

```suggestion
    @series.focal
    Scenario Outline: Attached enable of non-container services in a  container
        Given a `trusty` lxd container with ubuntu-advantage-tools installed
         when I attach contract_token with sudo
         and I run `ua enable <service>` as non-root
         then I will see the following on stderr:
            """
            This command must be run as root (try using sudo)
            """
         when I run `ua enable <service>` with sudo
         then I will see the following on stdout:
            """
            One moment, checking your subscription first
            Cannot install <title> on a container
            """

      Examples: Un-supported services in containers
        | service      | title        |
        | livepatch    | Livepatch    |
        | fips         | FIPS         |
        | fips-updates | FIPS Updates |
```

Reply:
I've addressed this one and will work on the WIP reduce duplication, PR 1047